### PR TITLE
test: re-enable skipped unit tests

### PR DESCRIPTION
**Pull request by MatissJanis**
Re-enabling and fixing skipped unit tests.

Before:
```
➤ YN0000: [loot-core]: Test Suites: 4 skipped, 32 passed, 32 of 36 total
➤ YN0000: [loot-core]: Tests:       33 skipped, 210 passed, 243 total
➤ YN0000: [loot-core]: Snapshots:   40 passed, 40 total
➤ YN0000: [loot-core]: Time:        131.646 s
```

After:
```
➤ YN0000: [loot-core]: Test Suites: 1 skipped, 35 passed, 35 of 36 total
➤ YN0000: [loot-core]: Tests:       2 skipped, 241 passed, 243 total
➤ YN0000: [loot-core]: Snapshots:   69 passed, 69 total
➤ YN0000: [loot-core]: Time:        145.029 s
```

### CODE REVIEW

**Comment by MatissJanis** [comment]
This PR almost fully solves: https://github.com/actualbudget/actual/issues/56

There are still 2x skipped tests, but those we can pick up next.

**Comment by biohzrddd** [comment]
Everything looks reasonable to me, small changes to get a lot of tests passing is awesome.

File: packages/loot-core/src/server/accounts/__snapshots__/transfer.test.js.snap [hunk 1]
```diff
@@ -424,13 +422,11 @@ exports[`Transfer transfers are properly inserted/updated/deleted 6`] = `
...
 +     \\"transfer_id\\": \\"id7\\",
-+   },
-+   Object {
+    },
+    Object {
       \\"account\\": \\"one\\",
       \\"amount\\": 5000,
-      \\"category\\": null,
-      \\"cleared\\": 1,
-      \\"date\\": 20170101,"
```
Comment (trevdor) [inline packages/loot-core/src/server/accounts/__snapshots__/transfer.test.js.snap:433]: I suppose something changed while this test suite was skipped. 
Do you know what that was that would explain this diff? (as well as the diffs at line 352 & 379)

**Review by trevdor — APPROVED** [review]
Yay for un-skipped tests! 🎉 
As long as you're confident in those snapshot changes I left a question about, ship it.

File: packages/loot-core/src/server/accounts/__snapshots__/transfer.test.js.snap (hunk 1 above)
Comment (MatissJanis) [reply packages/loot-core/src/server/accounts/__snapshots__/transfer.test.js.snap:433]: Sadly: no. And the git history only goes back as much as release of open source, so we might never find this out.

But IMO it's safe to merge it in the current state. Worst case scenario: we later discover that this test was faulty which forces us to fix it. Which still gives us a bit more security over having the test be disabled.

**Pull request merged**